docs/coverage-reporter/troubleshooting-coverage-cli-issues.md

```diff
@@ -10,14 +10,16 @@ The sections below provide instructions or workarounds to overcome common issues
 
 The Codacy Coverage Reporter automatically detects the SHA-1 hash of the current commit to associate with the coverage data when you're using one of the following CI/CD platforms:
 
--   <span class="skip-vale">Appveyor</span>
+<!-- vale off -->
```

Comment:
@prcr Vale keeps complaining. Suggestions?

Reply:
This is quite strange since the same syntax is working on other pages. :thinking: I tried running Vale locally and indeed it still detects the issues:

```bash
$ vale docs/*

 docs/coverage-reporter/uploading-coverage-in-advanced-scenarios.md
 66:41  error  Did you really mean 'golang'?  Vale.Spelling 


 docs/coverage-reporter/troubleshooting-coverage-cli-issues.md
 14:5    error  Did you really mean             Vale.Spelling 
                'Appveyor'?                                   
 17:5    error  Did you really mean 'Bitrise'?  Vale.Spelling 
 18:5    error  Did you really mean             Vale.Spelling 
                'Buildkite'?                                  
 20:5    error  Did you really mean             Vale.Spelling 
                'Codefresh'?                                  
 21:5    error  Did you really mean             Vale.Spelling 
                'Codeship'?                                   
 33:5    error  Did you really mean 'Wercker'?  Vale.Spelling 
 56:77   error  Did you really mean             Vale.Spelling 
                'detailedxml'?                                
 100:95  error  Did you really mean 'graal'?    Vale.Spelling 


 docs/coverage-reporter/index.md
 132:6   error  Did you really mean 'dariodf'?  Vale.Spelling 
 132:14  error  Did you really mean 'lcov'?     Vale.Spelling 
 133:6   error  Did you really mean             Vale.Spelling 
                'chrisgit'?                                   
 134:6   error  Did you really mean             Vale.Spelling 
                'danielpalme'?

[...]
```

Let's not waste time debugging Vale now (although, if you manage to replicate this consistently on a simpler Markdown file you could [open an issue on Vale's repository](https://github.com/errata-ai/vale/issues)), and revert to using the `<span class="skip-vale">...</span>` which seems to work. Sorry I made you lose some time around this!

Reply:
I did revert and open an issue. We'll see. https://github.com/errata-ai/vale/issues/529

Reply:
Nice, thanks for opening the issue!